frontend/src/App.js

```diff
@@ -21,6 +22,9 @@ const App = () => {
                   <Route path="/room/:roomId">
                      <RoomDetails />
                   </Route>
+                  <Route path="/users">
```

Comment:
I can access this route even without permission. A better option would be to create a protected route that will return a component if a user has permission to view it or redirect if the user isn't allowed to access the route. Here is a simple tutorial on how to do it: https://tylermcginnis.com/react-router-protected-routes-authentication/  
It also keeps components rendered in a protected route cleaner and independent of authorization as you don't need code like this:
```js
render() {
  return hasPermission ? <SomeComponent /> : null;
}
```

Reply:
Fixed in #88 - commit 244ee98 .